javascript-1/script4.js

```diff
@@ -0,0 +1,4 @@
+console.log("EXERCITIUL 4");
+let nume = "Serban";
+let prenume = "Andrei";
+console.log("Numeste meu este: " + nume + " " + prenume);
```

Comment:
Totul este ok, atata ca as propune aici sa modifici sa folosesti si un string tempalte literal sau interpolarea variabilelor in text
```js
console.log(`Numele meu este: ${nume} ${prenume}`);
```